composer.json

```diff
@@ -15,6 +15,9 @@
         "symfony/dependency-injection": "~2.7",
         "symfony/http-kernel": "~2.7"
     },
+    "require-dev": {
+        "phpunit/phpunit": "5.0.*"
```

Comment:
The bundle has to maintain PHP 5.4 compatibility, PHPUnit 5.0.* fails build because it supports php >= 5.6 https://travis-ci.org/skrz/autowiring-bundle/builds/87307234

It is better not to have a library depending on phpunit or other tools. Please don't add this.